torch/csrc/jit_decomp_interface.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <ATen/core/Tensor.h>
+#include <ATen/core/function_schema.h>
+#include <c10/macros/Export.h>
+
+namespace torch {
+namespace autograd {
```

Comment:
Since this is in the autograd namespace, maybe I want to move this file in to the torch/csrc/autograd directory

Reply:
No longer doing this